packages/cslam/include/cslam/LocalizationExperiment.py

```diff
@@ -234,7 +249,6 @@ def __callback__(self, msg, _):
                     if not self._graph.has_node(target_node_name):
```

Comment:
Is the chain of transforms that is added to the graph appended to the last optimized node or is it a chain added from the initial odometry node? It appears to me to be the former.

Reply:
I tested adding a random disturbance on the pose here and on every optimization the pose seems to reset to the correct one (i.e. the one discarding the previous odometry measurement)